lib/files.c

```diff
@@ -305,7 +307,27 @@ rpmfile_t *extract_rpm(struct rpminspect *ri, const char *pkg, Header hdr, const
         }
 
         /* Prepend output_dir to the path name */
-        xasprintf(&file_entry->fullpath, "%s/%s", *output_dir, archive_path);
+        tmp = archive_path;
```

Comment:
Will `joinpath` not do this?

Reply:
It should.  My initial run it didn't work, so I didn't think much of it.  joinpath.c comes from my secret project and I didn't want to dig in to it because I was already busy shaving this side of the yak.

Reply:
Oh right, I know why.  archive_path is from a libarchive structure and joinpath() wants allocated strings it can modify and walk independently.  It would work if I strdup()'ed them, but I just decided to do it this way instead.

Reply:
The stack, and all.